tests/apps/stateapp/app.go

```diff
@@ -239,7 +358,8 @@ func appRouter() *mux.Router {
 	router := mux.NewRouter().StrictSlash(true)
 
 	router.HandleFunc("/", indexHandler).Methods("GET")
-	router.HandleFunc("/test/{command}", handler).Methods("POST")
+	router.HandleFunc("/test/{command}", httpHandler).Methods("POST")
+	router.HandleFunc("/grpc-test", grpcHandler).Methods("POST")
```

Comment:
Could you please change the routing path like below ? you need to change http api go tests.

```
        router.HandleFunc("/test/http/{command}", httpHandler).Methods("POST")
	router.HandleFunc("/test/grpc/{command}", grpcHandler).Methods("POST")
```